.github/workflows/publish-docker-images.yaml

```diff
@@ -0,0 +1,94 @@
+name: Publish Docker Images
+
+on:
+  # `push` will build and public the image with a tag corresponding to the branch name (such as `main`, `master`).
+  # `push tag` will build and public the image with a tag has the same name as git tag (such as `v1.2.3`, `v0.0.1-alpha.1`) and a tag named `latest`.
+  push:
+    branches: [main, master]
```

Comment:
```suggestion
    branches: [main]
```

Reply:
Okay. This is just a measure of compatibility. I will remove them if not needed.